### ISSUE TITLE
react/jsx-no-leaked-render - multiple left operands yield false positive, in coerce moce

### ISSUE THREAD

**Issue by tulitheprogrammer** [jsx-eslint/eslint-plugin-react#3312]
that passes: 
```ts
{ !!someValue && <SomeElement /> }
```

but that isn't supported whatsoever:
```ts
{ !!someValue && !!someotherValue && <SomeElement /> }
```

**Comment by pke** [comment]
I think this is fixed in a recent PR but its not released to npm yet

**Comment by ljharb** [comment]
v7.30.1 is released; closing, but will reopen if it's not fixed.

**Comment by tulitheprogrammer** [comment]
fixed, just verified.
thanks !!!
